tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/getstarted-welcome.cy.js

```diff
@@ -0,0 +1,36 @@
+// <reference types="Cypress" />
+
+describe('Get Started Welcome Page', function () {
+
+    before(() => {
+        cy.visit('wp-admin/?page=nfd-onboarding&flow=ecommerce#/wp-setup/step/get-started/welcome');
+        cy.injectAxe(); 
```

Comment:
`cy.injectAxe()` does not do anything by itself. We need to add a `cy.checkA11y()` assertion somewhere. As David mentioned can we add this assertion and leave it commented out so that we can fix those later?

Reply:
added the checks and commented them out for now

Reply:
Let's remove `cy.injectAxe()` as well for now.